Task1/Task1.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>HTML Basics Task 1</title>
+    <link rel="stylesheet" type="text/css" href="Task1.css" />
+  </head>
+
+  <body>
+    <div class="informationContainer">
+      <div class="loginSection"><span class="loginText">Login</span></div>
+      <div class="informationBody">
+        <div class="aboutAndExperience">
+          <div class="about">
+            <span class="headerText">About</span>
+            <p class="description">
+              Having seen the IT industry grow exponentially, I could not help
+              myself held back and jumped into the industry as a trainee and
+              there was no going back. I have been worked as content writer,
+              data science Engineer and other major areas and then I tried
+              digging deep into frontend development. I am very good at Reactjs,
+              Typescript and JavaScript.
+            </p>
+          </div>
+          <div class="experience">
+            <span class="headerText">Experience</span>
+            <p class="description">Not required</p>
+          </div>
+        </div>
+        <div class="profileSection">
+          <img
```

Comment:
Add an alt attribute to the <img> tag to describe the image, which improves accessibility for screen readers.